cpp/models/abm/person.cpp

```diff
@@ -137,6 +140,24 @@ bool Person::goes_to_work(TimePoint t, const AbmMigrationParameters& params) con
     return m_random_workgroup < params.get<WorkRatio>().get_matrix_at(t.days())[0];
 }
 
+TimeSpan Person::get_go_to_work_time(const AbmMigrationParameters& params, TimeSpan dt) const
+{
+    TimeSpan minimum_goto_work_time = params.get<GotoWorkTimeMinimum>()[m_age];
+    TimeSpan maximum_goto_work_time = params.get<GotoWorkTimeMaximum>()[m_age];
+    int timeSlots = (maximum_goto_work_time.seconds() - minimum_goto_work_time.seconds()) / dt.seconds();
+    int seconds_after_minimum = int (timeSlots * m_random_goto_work_hour) * dt.seconds();
+    return minimum_goto_work_time+ mio::seconds(seconds_after_minimum);
```

Comment:
I don't quite understand why you need to use dt here. doesn't it cancel out?

Reply:
with the new changes that I did above I also could change this method. So it does not have the dt anymore!